### ISSUE TITLE
Python 3

### ISSUE THREAD

**Issue by 0x00B1** [CellProfiler/CellProfiler#1559]
- [ ] Migrate wxPython to wxPython-Phoenix

**Comment by 0x00B1** [comment]
```sh
$ pip freeze
altgraph==0.12
args==0.1.0
cellh5==1.2.0
CellProfiler==0.0.0
clint==0.4.1
Cython==0.23
fix-osx-virtualenv==0.1
funcsigs==0.4
future==0.15.0
h5py==2.5.0
javabridge==1.0.11
libtiff==0.4.0
lxml==3.4.4
macholib==1.7
matplotlib==1.4.3
mock==1.3.0
modulegraph==0.12.1
MySQL-python==1.2.5
node==0.9.15
nose==1.3.7
numpy==1.9.2
odict==1.5.1
pandas==0.16.2
pbr==1.5.0
pluggy==0.3.0
plumber==1.3
py==1.4.30
py2app==0.9
pyfaidx==0.4.2
pyparsing==2.0.3
pytest==2.7.2
python-bioformats==1.0.5
python-dateutil==2.4.2
pytz==2015.4
pyzmq==14.7.0
requests==2.7.0
scikit-learn==0.16.1
scipy==0.16.0
six==1.9.0
sklearn==0.0
tox==2.1.1
virtualenv==13.1.2
wheel==0.24.0
wxPython==3.0.2.0
zmq==0.0.0
zope.component==4.2.2
zope.deprecation==4.1.2
zope.event==4.0.3
zope.interface==4.1.2
zope.lifecycleevent==4.1.0
```

**Comment by 0x00B1** [comment]
Closing to replace with smaller Python 3 related issues
